sympy/polys/polytools.py

```diff
@@ -1979,6 +1978,9 @@ def coeff(f, x, n=1, right=False):
     def LM(f, order=None):
         """
         Returns the leading monomial of ``f``.
+	
```

Comment:
inconsistent use of tabs and spaces in indentation. Use [PEP](https://packagecontrol.io/packages/Python%20PEP8%20Autoformat) plugin to cross check python code format.